src/loan-options/special-loan-programs/index.html

```diff
@@ -12,7 +12,7 @@
 
       <h2>Special Loan Progams</h2>
 
-      <p>For people who qualify, special loan programs can be more affordable than a <a href="../conventional-loans/" target="_blank">conventional</a> or <a href="../FHA-loans/" target="_blank">FHA</a> loan, so make sure to check to see if you are eligible. Always compare official loan offers, called <a href="/askcfpb/146/what-is-a-good-faith-estimate-what-is-a-gfe.html" target="_blank">Good Faith Estimates</a>, before making a final decision.</p>
+      <p>For people who qualify, special loan programs can be more affordable than a <a href="../conventional-loans/" target="_blank">conventional</a> or <a href="../FHA-loans/" target="_blank">FHA</a> loan, so make sure to check to see if you are eligible. Always compare official loan offers, called <a href="http://consumerfinance.gov/askcfpb/1995/what-is-a-loan-estimate.html" target="_blank">Loan Estimates</a>, before making a final decision.</p>
```

Comment:
Link changes to this page require browser test updates here https://github.com/cfpb/owning-a-home/blob/master/test/browser_testing/features/loan_special.feature